e2e-tests/ukef/cypress/e2e/journeys/portal/deal-cancellation/BSS-EWCS/deal-cancellation-maker-unable-issue-facility.spec.js

```diff
@@ -0,0 +1,84 @@
+import portalPages from '../../../../../../../portal/cypress/e2e/pages';
+import MOCK_USERS from '../../../../../../../e2e-fixtures/portal-users.fixture';
+import getAinUnissuedDeals from '../../test-data/AIN-deal-unissued-facilities/dealReadyToSubmit';
+import getMinUnissuedDeals from '../../test-data/MIN-deal-unissued-facilities/dealReadyToSubmit';
+import { TFM_URL, PIM_USER_1 } from '../../../../../../../e2e-fixtures';
+
+import { yesterday, tomorrow } from '../../../../../../../e2e-fixtures/dateConstants';
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+context('Deal cancellation', () => {
+  const ainUnissuedDeals = Array(2).fill(getAinUnissuedDeals());
+  const minUnissuedDeals = Array(2).fill(getMinUnissuedDeals());
+  let deals = [];
+  const dealFacilities = [];
+
+  before(() => {
+    cy.insertManyDeals([...ainUnissuedDeals, ...minUnissuedDeals], BANK1_MAKER1).then((insertedDeals) => {
+      insertedDeals.forEach((insertedDeal) => {
+        cy.createFacilities(insertedDeal._id, insertedDeal.mockFacilities, BANK1_MAKER1).then((createdFacilities) => {
+          dealFacilities.push(...createdFacilities);
+        });
+      });
+      deals = [...insertedDeals];
+    });
+  });
+
+  beforeEach(() => {
+    cy.clearCookie('dtfs-session');
+    cy.clearCookie('_csrf');
+    cy.getCookies().should('be.empty');
+    cy.login(BANK1_MAKER1);
+  });
+
+  after(() => {
+    cy.clearCookies();
+    cy.clearCookie('dtfs-session');
+    cy.clearCookie('_csrf');
+    cy.getCookies().should('be.empty');
+  });
+
+  describe('Deal cancellations on tfm with effective dates in the past and in the future', () => {
```

Comment:
Could you split this into two describe blocks:

```
describe('when the effective date is in the past', () => {

});

describe('when the effective date is in the future', () => {

});
```

then the `it` descriptions inside each wont need to mention the effective date.